examples/standard/main.tf

```diff
@@ -78,8 +71,8 @@ locals {
       }
     },
     {
```

Comment:
Perhaps just remove the second worker pool here in this example to simplify and make easier to comprehend.

That standard example would use only one single worker pool (the default one) across the 3 AZs